IDEAS/Fluid/BaseCircuits/FluidCollector.mo

```diff
@@ -0,0 +1,50 @@
+within IDEAS.Fluid.BaseCircuits;
```

Comment:
@rubenbaetens you forgot to add this file in package.order
can you fix this? thanks!